build/yamls/base/conf/antrea-controller.conf

```diff
@@ -53,3 +53,22 @@ featureGates:
 # API groups. After adding the annotation, legacy CRDs can be deleted safely without impacting
 # new CRDs.
 #legacyCRDMirroring: true
+
+# Enable the integrated node IPAM controller within the Antrea controller.
+#enableNodeIPAMController: false
+
+# Cluster IP CIDR ranges for Pods. Required when enableNodeIPAMController is enabled, and should be set with
+# the value value specified by --cluster-cidr for kube-apiserver.
+#clusterCIDRs: 172.18.0.0/16
+
+# Service IP CIDR range for Services. Required when enableNodeIPAMController is enabled, and should be set with
+# the value value specified by --service-cluster-ip-range for kube-apiserver.
+#serviceCIDR: 172.19.0.0/16
+
+# Node CIDR mask size for IPv4. Required when enableNodeIPAMController is enabled, and should be set with
+# the value value specified by --node-cidr-mask-size-ipv4 for kube-controller-manager.
```

Comment:
Doesn't we assume NodeIPAMController of kube-controller-manager is not enabled when antrea enables it? If both are enabled, could there be conflicts to update the node CIDR?

Reply:
If both are enabled, they would race for updating the node spec. The value should be same value that would be set for kube-controller-manager when NodeIPAM is enabled - as this is the same NodeIPAM.

Reply:
I think current comment will make user think they should also set the same value to kube-controller-manager. But we don't require it, right? Maybe the comment doesn't need to mention kube-controller-manager.